Example/UITests/VOKNavigationHelper_UITests.m

```diff
@@ -0,0 +1,42 @@
+//
+//  VOKNavigationHelper_UITests.m
+//  VOKUtilities
+//
+//  Created by Brock Boland on 1/24/17.
+//  Copyright © 2017 Vokal. All rights reserved.
+//
+
+#import <KIF/KIF.h>
+#import "VOKNavigationHelper.h"
+#import "AnotherViewController.h"
+#import "TestViewController.h"
+
+@interface VOKNavigationHelper_UITests : KIFTestCase
+
+@end
+
+@implementation VOKNavigationHelper_UITests
+
+- (void)afterAll
+{
+    // Reset back to the default view controller
+    [VOKNavigationHelper clearExistingViewsAndSwitchToViewController:[[TestViewController alloc] init]];
+    [super afterAll];
+}
+
+- (void)testStoryboardJumping
+{
+    [VOKNavigationHelper clearExistingViewsAndSwitchToStoryboardNamed:@"First"];
+    [tester waitForViewWithAccessibilityLabel:@"This is in the first storyboard"];
+    [VOKNavigationHelper clearExistingViewsAndSwitchToStoryboardNamed:@"Second"];
+    [tester waitForViewWithAccessibilityLabel:@"This is in the second storyboard"];
```

Comment:
might want to just make these properties so you're referencing the same string in both places